app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -1,6 +1,10 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < ManageIQ::Providers::EmbeddedAutomationManager::Authentication
   FRIENDLY_NAME = "Embedded Ansible Credential".freeze
 
+  def self.credential_type
+    "embedded_ansible_credential_types"
```

Comment:
I'm wondering if we even need this method.  It acts as a key into a hash that's presented over the API, but
a) maybe we don't need the key at all
b) if we do need some sort of key, just use the class name
c) if we do need the key and it can't be the class name, then just derive the key with something like `"#{klass.name.gsub("::", "_").underscore}_credential_types`

Reply:
I think https://github.com/ManageIQ/manageiq/pull/22995/files#r1567779476 answers this, we need it for now because of how the UI splits external / internal automation controllers up ignoring the actual EMS object and assuming there is only one of the internal types.